packages/core/index.d.ts

```diff
@@ -215,6 +215,7 @@ declare module '@rjsf/core' {
             name: string;
             disabled: boolean;
             readonly: boolean;
+            hidden: boolean;
```

Comment:
Can you update the docs as well where we document the props passed over to ObjectFieldTemplate?

Reply:
Done

Reply:
Thanks!